src/main/java/org/fcrepo/spec/testsuite/authz/WebACDefaultACLs.java

```diff
@@ -0,0 +1,134 @@
+import io.restassured.response.Response;
+import org.fcrepo.spec.testsuite.TestInfo;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+/**
+ * @author dbernstein
+ * @since 2018-09-17
+ */
+public class WebACDefaultACLs extends AbstractAuthzTest {
+
+    /**
+     * Constructor
+     *
+     * @param adminUsername admin username
+     * @param adminPassword admin password
+     * @param username      username
+     * @param password      password
+     */
+    @Parameters({"param2", "param3", "param4", "param5"})
+    public WebACDefaultACLs(final String adminUsername, final String adminPassword, final String username,
+                            final String password) {
+        super(adminUsername, adminPassword, username, password);
+    }
+
+    /**
+     * 5.9-A - Inheritance of ACLs in Fedora implementations is defined by the [SOLIDWEBAC]
+     * ACL Inheritance Algorithm and must be reckoned along the [LDP] containment
+     * relationships linking controlled resources
+     *
+     * @param uri of base fedora uri
+     */
+    @Test(groups = {"MUST"})
+    @Parameters({"param0"})
+    public void aclInheritanceMustUseLdpContainmentRelationships(final String uri) {
+        final TestInfo info =
+            setupTest("5.9-A", "Inheritance of ACLs in Fedora implementations is defined by the [SOLIDWEBAC]" +
+                               "ACL Inheritance Algorithm and must be reckoned along the [LDP] containment " +
+                               "relationships linking controlled resources",
+                      "https://fedora.info/2018/06/25/spec/#inheritance", ps);
+        final String simpleRDFBody = "@prefix dc: <http://purl.org/dc/terms/> \n" +
+                                     "<> dc:title \"Test\".";
+        final Headers headers = new Headers(new Header("Content-Type", "text/turtle"));
+
+        //create a resource
+        final String grandParentUri = createResource(uri, info.getId());
+        //create an acl with acl:default with write privileges
+        createAclForResource(grandParentUri, "user-read-write.ttl", this.username);
+        //create a child  and child acl with read privileges and no acl:default
+        final Response childPost = doPost(grandParentUri, headers, simpleRDFBody);
+        final String childUri = getLocation(childPost);
+        createAclForResource(childUri, "user-read-only-no-default.ttl", this.username);
+
+        //create a grandchild  with no acl.
+        final Response grandChildPost = doPost(childUri, headers, simpleRDFBody);
+        final String grandChildUri = getLocation(grandChildPost);
+
+        // verify that the user can read and write to the grandparent
+        doGet(grandParentUri, false);
+        doPost(grandParentUri, headers, simpleRDFBody, false);
+
+        // verify that the user can read but not write to the child
+        doPostUnverified(childUri, headers, simpleRDFBody, false).then().statusCode(clientErrorRange());
```

Comment:
It doesn't seem like we're totally consistent within the authz tests, but perhaps this check should be for 403 specifically? It seems that we want to ensure it is an authorization failure versus other bad requests, and that code should be pretty consistent across impls.

Reply:
good point.